README.md

```diff
@@ -56,6 +56,15 @@ The above workflow (when manually triggered) will
 ![image](https://user-images.githubusercontent.com/2510597/140506212-4938e44d-0662-4dc5-9fb1-c3f59fe075a6.png)
 
 - When you merge this PR, it will request an Npm OTP from Optic. (If you close the PR, nothing will happen)
+- You need to define Npm and Optic tokens in the Github secrets for each user that will receive the otp
+- You can either provide npm and optic tokens of the user who has initiated the workflow or provide default tokens or both.  
+**Some examples**:  
+  - Use only default tokens:   
+    *e.g.* npm-token: ${{secrets.NPM_TOKEN}}
```

Comment:
```suggestion
    *e.g.* npm-token: ${{ secrets.NPM_TOKEN }}
```

Reply:
@sameer-coder please address this comment (simply accept the suggestion)